Tasks/Otkidach/task1.py

```diff
@@ -0,0 +1,10 @@
+import math
+import decimal
+from decimal import Decimal
+
+dep = 20000
+perc = 15
+term = 5
+summ = decimal.Decimal(dep*math.pow((1+perc/(100*12)),term*12))
+summ.quantize(Decimal('1.00')) #почему не происходит округления?... (╮°-°)╮┳━━┳ ( ╯°□°)╯ ┻━━┻
```

Comment:
the rounding works, but the **quantize** function actually returns new rounded value instead of modifying the existing one, so you just need to store it a new variable (or pass directly to **print**)

appreciate the ASCII art by the way))